Description: spelling-error-in-binary
 Lintian found a spelling error in the given binary. Lintian has a list of common
 misspellings that it looks for. It does not have a dictionary like a spelling
 checker does.
Origin: http://lintian.debian.org/tags/spelling-error-in-binary.html

Index: kwave-0.8.11-1/plugins/codec_ogg/OpusDecoder.cpp
===================================================================
--- kwave-0.8.11-1.orig/plugins/codec_ogg/OpusDecoder.cpp	2013-10-20 23:38:30.000000000 +1100
+++ kwave-0.8.11-1/plugins/codec_ogg/OpusDecoder.cpp	2013-12-08 09:23:01.000000000 +1100
@@ -159,9 +159,9 @@
 	const unsigned char *c   = m_op.packet;
 	unsigned long int length = m_op.bytes;
 
-	// check lenght of comments and magic value
+	// check length of comments and magic value
 	if (length < 16) {
-	    qWarning("OpusDecoder::parseHeader(): comment lenght < 16 (%lu)",
+	    qWarning("OpusDecoder::parseHeader(): comment length < 16 (%lu)",
 	              length);
 	    break;
 	}
